ipfx/data_set_features.py

```diff
@@ -308,21 +339,23 @@ def extract_data_set_features(data_set, subthresh_min_amp=None):
             logging.info("Assigned subthreshold minimum amplitude of %f.", subthresh_min_amp)
 
 
-    cell_features = extract_cell_features(data_set,
-                                          ramp_sweep_numbers,
-                                          ssq_sweep_numbers,
-                                          lsq_sweep_numbers,
-                                          subthresh_min_amp)
-
     # compute sweep features
     sweep_features = extract_sweep_features(data_set, iclamp_sweeps)
 
+    # try:
+    cell_features, cell_state = extract_cell_features(data_set,
+                                        ramp_sweep_numbers,
+                                        ssq_sweep_numbers,
+                                        lsq_sweep_numbers,
+                                        subthresh_min_amp)
+    # TODO: looks like this is adding peak_deflect, but could be refactored not to need cell_features
     # shuffle peak deflection for the subthreshold long squares
-    for s in cell_features["long_squares"]["subthreshold_sweeps"]:
-        sweep_features[s['sweep_number']]['peak_deflect'] = s['peak_deflect']
-
-    cell_record = fr.build_cell_feature_record(cell_features)
+    if cell_features["long_squares"]:
+        for s in cell_features["long_squares"]["subthreshold_sweeps"]:
```

Comment:
This looks again like an already present issue, but we should not use single-letter variables.